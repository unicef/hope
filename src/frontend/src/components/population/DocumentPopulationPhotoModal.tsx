import * as React from 'react';
import { IndividualNode, useIndividualPhotosQuery } from '@generated/graphql';
import { PhotoModal } from '@core/PhotoModal/PhotoModal';

interface DocumentPopulationPhotoModalProps {
  individual: IndividualNode;
  documentNumber: string;
  documentId: string;
}

export function DocumentPopulationPhotoModal({
  individual,
  documentNumber,
  documentId,
}: DocumentPopulationPhotoModalProps): React.ReactElement {
  const { data } = useIndividualPhotosQuery({
    variables: { id: individual?.id },
    fetchPolicy: 'network-only',
  });
  const documentWithPhoto = data?.individual?.documents?.edges?.find(
    (el) => el.node.id === documentId,
  );

  return (
    <PhotoModal
      src={documentWithPhoto?.node?.photo}
      linkText={documentNumber}
      variant="link"
      title="Document's Photo"
    />
  );
}
