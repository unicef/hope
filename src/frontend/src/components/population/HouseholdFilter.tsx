import { Grid, MenuItem } from '@mui/material';
import AssignmentIndRoundedIcon from '@mui/icons-material/AssignmentIndRounded';
import FlashOnIcon from '@mui/icons-material/FlashOn';
import GroupIcon from '@mui/icons-material/Group';
import * as React from 'react';
import { useTranslation } from 'react-i18next';
import { useLocation, useNavigate } from 'react-router-dom';
import { HouseholdChoiceDataQuery, ProgramNode } from '@generated/graphql';
import { useBaseUrl } from '@hooks/useBaseUrl';
import { AdminAreaAutocomplete } from '@shared/autocompletes/AdminAreaAutocomplete';
import { householdTableOrderOptions } from '@utils/constants';
import { createHandleApplyFilterChange } from '@utils/utils';
import { FiltersSection } from '@core/FiltersSection';
import { NumberTextField } from '@core/NumberTextField';
import { SearchTextField } from '@core/SearchTextField';
import { SelectFilter } from '@core/SelectFilter';
import { DocumentSearchField } from '@core/DocumentSearchField';

interface HouseholdFiltersProps {
  filter;
  programs?: ProgramNode[];
  choicesData: HouseholdChoiceDataQuery;
  setFilter: (filter) => void;
  initialFilter;
  appliedFilter;
  setAppliedFilter: (filter) => void;
  isOnPaper?: boolean;
}

export function HouseholdFilters({
  filter,
  programs,
  choicesData,
  setFilter,
  initialFilter,
  appliedFilter,
  setAppliedFilter,
  isOnPaper = true,
}: HouseholdFiltersProps): React.ReactElement {
  const { t } = useTranslation();
  const navigate = useNavigate();
  const location = useLocation();
  const { isAllPrograms } = useBaseUrl();
  const { handleFilterChange, applyFilterChanges, clearFilter } =
    createHandleApplyFilterChange(
      initialFilter,
      navigate,
      location,
      filter,
      setFilter,
      appliedFilter,
      setAppliedFilter,
    );
  const handleApplyFilter = (): void => {
    applyFilterChanges();
  };

  const handleClearFilter = (): void => {
    clearFilter();
  };

  return (
    <FiltersSection
      clearHandler={handleClearFilter}
      applyHandler={handleApplyFilter}
      isOnPaper={isOnPaper}
    >
      <Grid container alignItems="flex-end" spacing={3}>
        <Grid item xs={3}>
          <SearchTextField
            label={t('Search')}
            value={filter.search}
            onChange={(e) => handleFilterChange('search', e.target.value)}
            data-cy="hh-filters-search"
          />
        </Grid>
        <DocumentSearchField
          onChange={handleFilterChange}
          type={filter.documentType}
          number={filter.documentNumber}
          choices={choicesData?.documentTypeChoices}
        />
        {isAllPrograms && (
          <Grid item xs={3}>
            <SelectFilter
              onChange={(e) => handleFilterChange('program', e.target.value)}
              label={t('Programme')}
              value={filter.program}
              fullWidth
              icon={<FlashOnIcon />}
              data-cy="hh-filters-program"
            >
              {programs.map((program) => (
                <MenuItem key={program.id} value={program.id}>
                  {program.name}
                </MenuItem>
              ))}
            </SelectFilter>
          </Grid>
        )}
        <Grid item xs={3}>
          <SelectFilter
            onChange={(e) =>
              handleFilterChange('residenceStatus', e.target.value)
            }
            label={t('Residence Status')}
            fullWidth
            value={filter.residenceStatus}
            icon={<AssignmentIndRoundedIcon />}
            data-cy="hh-filters-residence-status"
          >
            {choicesData.residenceStatusChoices?.map((status) => (
              <MenuItem key={status.value} value={status.value}>
                {status.name}
              </MenuItem>
            ))}
          </SelectFilter>
        </Grid>
        <Grid item xs={3}>
          <NumberTextField
            topLabel={t('Household Size')}
            value={filter.householdSizeMin}
            placeholder={t('From')}
            icon={<GroupIcon />}
            fullWidth
            onChange={(e) =>
              handleFilterChange('householdSizeMin', e.target.value)
            }
            data-cy="hh-filters-household-size-from"
          />
        </Grid>
        <Grid item xs={3}>
          <NumberTextField
            value={filter.householdSizeMax}
            placeholder={t('To')}
            icon={<GroupIcon />}
            fullWidth
            onChange={(e) =>
              handleFilterChange('householdSizeMax', e.target.value)
            }
            data-cy="hh-filters-household-size-to"
          />
        </Grid>
        <Grid item xs={3}>
          <SelectFilter
            onChange={(e) => handleFilterChange('orderBy', e.target.value)}
            label={t('Sort by')}
            value={filter.orderBy}
            data-cy="hh-filters-order-by"
            disableClearable
          >
            {householdTableOrderOptions.map((order) => (
              <MenuItem key={order.value} value={order.value}>
                {order.name}
              </MenuItem>
            ))}
          </SelectFilter>
        </Grid>
        <Grid item xs={3}>
          <SelectFilter
            onChange={(e) => handleFilterChange('withdrawn', e.target.value)}
            label={t('Status')}
            value={filter.withdrawn}
            data-cy="hh-filters-status"
          >
            <MenuItem key="all" value="null">
              All
            </MenuItem>
            <MenuItem key="active" value="false">
              Active
            </MenuItem>
            <MenuItem key="inactive" value="true">
              Withdrawn
            </MenuItem>
          </SelectFilter>
        </Grid>
        {isAllPrograms && (
          <Grid item xs={3}>
            <SelectFilter
              onChange={(e) =>
                handleFilterChange('programState', e.target.value)
              }
              label={t('Programme State')}
              value={filter.programState}
              fullWidth
              disableClearable
              data-cy="filters-program-state"
            >
              <MenuItem value="active">{t('Active Programmes')}</MenuItem>
              <MenuItem value="all">{t('All Programmes')}</MenuItem>
            </SelectFilter>
          </Grid>
        )}
        <Grid item xs={3}>
          <AdminAreaAutocomplete
            name="admin1"
            level={1}
            value={filter.admin1}
            filter={filter}
            setFilter={setFilter}
            initialFilter={initialFilter}
            appliedFilter={appliedFilter}
            setAppliedFilter={setAppliedFilter}
            disabled={filter.admin2}
            dataCy="hh-filters-admin1"
          />
        </Grid>
        <Grid item xs={3}>
          <AdminAreaAutocomplete
            name="admin2"
            level={2}
            value={filter.admin2}
            filter={filter}
            setFilter={setFilter}
            initialFilter={initialFilter}
            appliedFilter={appliedFilter}
            setAppliedFilter={setAppliedFilter}
            disabled={filter.admin1}
            dataCy="hh-filters-admin2"
          />
        </Grid>
      </Grid>
    </FiltersSection>
  );
}
