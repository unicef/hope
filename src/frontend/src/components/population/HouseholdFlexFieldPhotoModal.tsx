import { Box } from '@mui/material';
import * as React from 'react';
import { useParams } from 'react-router-dom';
import { useHouseholdFlexFieldsQuery } from '@generated/graphql';
import { PhotoModal } from '@core/PhotoModal/PhotoModal';

export function HouseholdFlexFieldPhotoModal({ field }): React.ReactElement {
  const { id } = useParams();
  const { data } = useHouseholdFlexFieldsQuery({
    variables: { id },
    fetchPolicy: 'network-only',
  });

  if (!data) {
    return null;
  }

  const { flexFields } = data.household;
  const picUrl = flexFields[field.name];

  return picUrl ? (
    <PhotoModal src={picUrl} />
  ) : (
    <Box display="flex" alignItems="center">
      -
    </Box>
  );
}
