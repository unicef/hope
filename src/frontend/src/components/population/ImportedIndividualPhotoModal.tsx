import * as React from 'react';
import {
  ImportedIndividualNode,
  useImportedIndividualPhotosQuery,
} from '@generated/graphql';
import { PhotoModal } from '@core/PhotoModal/PhotoModal';

interface ImportedIndividualPhotoModalProps {
  individual: ImportedIndividualNode;
}

export function ImportedIndividualPhotoModal({
  individual,
}: ImportedIndividualPhotoModalProps): React.ReactElement {
  const { data } = useImportedIndividualPhotosQuery({
    variables: { id: individual?.id },
    fetchPolicy: 'network-only',
  });

  return (
    <PhotoModal
      src={data?.importedIndividual?.photo}
      variant="button"
      title="Individuals's Photo"
    />
  );
}
