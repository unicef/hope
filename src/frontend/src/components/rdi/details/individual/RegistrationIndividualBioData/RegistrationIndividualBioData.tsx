import { Box, Grid, Paper, Typography } from '@mui/material';
import * as React from 'react';
import { useTranslation } from 'react-i18next';
import styled from 'styled-components';
import {
  choicesToDict,
  formatAge,
  getPhoneNoLabel,
  renderBoolean,
  sexToCapitalize,
} from '@utils/utils';
import {
  HouseholdChoiceDataQuery,
  ImportedIndividualDetailedFragment,
} from '@generated/graphql';
import { ContentLink } from '@core/ContentLink';
import { LabelizedField } from '@core/LabelizedField';
import { Title } from '@core/Title';
import { UniversalMoment } from '@core/UniversalMoment';
import { DocumentRegistrationPhotoModal } from '../DocumentRegistrationPhotoModal';
import { useBaseUrl } from '@hooks/useBaseUrl';

const Overview = styled(Paper)`
  padding: ${({ theme }) => theme.spacing(8)}
    ${({ theme }) => theme.spacing(11)};
`;

const BorderBox = styled.div`
  border-bottom: 1px solid #e1e1e1;
`;

interface RegistrationIndividualBioDataProps {
  individual: ImportedIndividualDetailedFragment;
  choicesData: HouseholdChoiceDataQuery;
}

export function RegistrationIndividualBioData({
  individual,
  choicesData,
}: RegistrationIndividualBioDataProps): React.ReactElement {
  const { baseUrl } = useBaseUrl();
  const { t } = useTranslation();
  const relationshipChoicesDict = choicesToDict(
    choicesData.relationshipChoices,
  );
  const maritalStatusChoicesDict = choicesToDict(
    choicesData.maritalStatusChoices,
  );
  const workStatusChoicesDict = choicesToDict(choicesData.workStatusChoices);
  const observedDisabilityChoicesDict = choicesToDict(
    choicesData.observedDisabilityChoices,
  );
  const severityOfDisabilityChoicesDict = choicesToDict(
    choicesData.severityOfDisabilityChoices,
  );
  const roleChoicesDict = choicesToDict(choicesData.roleChoices);
  const mappedIndividualDocuments = individual.documents?.edges?.map((edge) => (
    <Grid key={edge.node.id} item xs={3}>
      <Box flexDirection="column">
        <Box mb={1}>
          <LabelizedField label={edge.node.type.label}>
            {edge.node.photo ? (
              <DocumentRegistrationPhotoModal
                documentNumber={edge.node.documentNumber}
                documentId={edge.node.id}
                individual={individual}
              />
            ) : (
              edge.node.documentNumber
            )}
          </LabelizedField>
        </Box>
        <LabelizedField label={t('issued')}>{edge.node.country}</LabelizedField>
      </Box>
    </Grid>
  ));

  const mappedIdentities = individual.identities?.edges?.map((item) => (
    <Grid key={item.node.id} item xs={3}>
      <Box flexDirection="column">
        <Box mb={1}>
          <LabelizedField label={`${item.node.partner} ID`}>
            {item.node.number}
          </LabelizedField>
        </Box>
        <LabelizedField label={t('issued')}>{item.node.country}</LabelizedField>
      </Box>
    </Grid>
  ));

  return (
    <Overview>
      <Title>
        <Typography variant="h6">{t('Bio Data')}</Typography>
      </Title>
      <Grid container spacing={6}>
        <Grid item xs={3}>
          <LabelizedField label={t('Full Name')}>
            {individual.fullName}
          </LabelizedField>
        </Grid>
        <Grid item xs={3}>
          <LabelizedField label={t('Given Name')}>
            {individual.givenName}
          </LabelizedField>
        </Grid>
        <Grid item xs={3}>
          <LabelizedField label={t('Middle Name')}>
            {individual.middleName}
          </LabelizedField>
        </Grid>
        <Grid item xs={3}>
          <LabelizedField label={t('Family Name')}>
            {individual.familyName}
          </LabelizedField>
        </Grid>
        <Grid item xs={3}>
          <LabelizedField label={t('Gender')}>
            {sexToCapitalize(individual.sex)}
          </LabelizedField>
        </Grid>
        <Grid item xs={3}>
          <LabelizedField label={t('Age')}>
            {formatAge(individual.age)}
          </LabelizedField>
        </Grid>
        <Grid item xs={3}>
          <LabelizedField label={t('Date of Birth')}>
            <UniversalMoment>{individual.birthDate}</UniversalMoment>
          </LabelizedField>
        </Grid>
        <Grid item xs={3}>
          <LabelizedField label={t('Estimated Date of Birth')}>
            {renderBoolean(individual.estimatedBirthDate)}
          </LabelizedField>
        </Grid>
        <Grid item xs={3}>
          <LabelizedField label={t('Marital Status')}>
            {maritalStatusChoicesDict[individual.maritalStatus]}
          </LabelizedField>
        </Grid>
        <Grid item xs={3}>
          <LabelizedField label={t('Work Status')}>
            {workStatusChoicesDict[individual.workStatus]}
          </LabelizedField>
        </Grid>
        <Grid item xs={3}>
          <LabelizedField label={t('Pregnant')}>
            {renderBoolean(individual.pregnant)}
          </LabelizedField>
        </Grid>
        <Grid item xs={3}>
          <LabelizedField label={t('Household ID')}>
            {individual?.household?.id ? (
              <ContentLink
                href={`/${baseUrl}/registration-data-import/household/${individual?.household?.id}`}
              >
                {individual?.household.importId}
              </ContentLink>
            ) : (
              '-'
            )}
          </LabelizedField>
        </Grid>
        <Grid item xs={3}>
          <LabelizedField label={t('Role')}>
            {roleChoicesDict[individual.role]}
          </LabelizedField>
        </Grid>
        <Grid item xs={3}>
          <LabelizedField label={t('Relationship to HOH')}>
            {relationshipChoicesDict[individual.relationship]}
          </LabelizedField>
        </Grid>
        <Grid item xs={3}>
          <LabelizedField label={t('Preferred language')}>
            {individual.preferredLanguage}
          </LabelizedField>
        </Grid>
        <Grid item xs={12}>
          <BorderBox />
        </Grid>
        <Grid item xs={3}>
          <LabelizedField label={t('Observed disabilities')}>
            {individual.observedDisability
              .map((choice) => observedDisabilityChoicesDict[choice])
              .join(', ')}
          </LabelizedField>
        </Grid>
        <Grid item xs={3}>
          <LabelizedField label={t('Seeing disability severity')}>
            {severityOfDisabilityChoicesDict[individual.seeingDisability]}
          </LabelizedField>
        </Grid>
        <Grid item xs={3}>
          <LabelizedField label={t('Hearing disability severity')}>
            {severityOfDisabilityChoicesDict[individual.hearingDisability]}
          </LabelizedField>
        </Grid>
        <Grid item xs={3}>
          <LabelizedField label={t('Physical disability severity')}>
            {severityOfDisabilityChoicesDict[individual.physicalDisability]}
          </LabelizedField>
        </Grid>
        <Grid item xs={3}>
          <LabelizedField
            label={t('Remembering or concentrating disability severity')}
          >
            {severityOfDisabilityChoicesDict[individual.memoryDisability]}
          </LabelizedField>
        </Grid>
        <Grid item xs={3}>
          <LabelizedField label={t('Self-care disability severity')}>
            {severityOfDisabilityChoicesDict[individual.selfcareDisability]}
          </LabelizedField>
        </Grid>
        <Grid item xs={3}>
          <LabelizedField label={t('Communicating disability severity')}>
            {severityOfDisabilityChoicesDict[individual.commsDisability]}
          </LabelizedField>
        </Grid>
        <Grid item xs={3}>
          <LabelizedField label={t('Disability')}>
            {individual.disability === 'DISABLED' ? 'Disabled' : 'Not Disabled'}
          </LabelizedField>
        </Grid>
        {!mappedIndividualDocuments?.length &&
        !mappedIdentities.length ? null : (
          <Grid item xs={12}>
            <BorderBox />
          </Grid>
        )}
        {mappedIndividualDocuments}
        {mappedIdentities}
        <Grid item xs={12}>
          <BorderBox />
        </Grid>
        <Grid item xs={3}>
          <LabelizedField label={t('Email')}>
            {individual?.email}
          </LabelizedField>
        </Grid>
        <Grid item xs={3}>
          <LabelizedField label={t('Phone Number')}>
            {getPhoneNoLabel(individual.phoneNo, individual.phoneNoValid)}
          </LabelizedField>
        </Grid>
        <Grid item xs={3}>
          <LabelizedField label={t('Alternate Phone Number')}>
            {getPhoneNoLabel(
              individual.phoneNoAlternative,
              individual.phoneNoAlternativeValid,
            )}
          </LabelizedField>
        </Grid>
      </Grid>
    </Overview>
  );
}
