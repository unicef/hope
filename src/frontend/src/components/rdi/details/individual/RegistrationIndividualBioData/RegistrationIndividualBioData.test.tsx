import * as React from 'react';
import { fakeHouseholdChoices } from '../../../../../../fixtures/population/fakeHouseholdChoices';
import { fakeImportedIndividualDetailedFragment } from '../../../../../../fixtures/registration/fakeImportedIndividualDetailedFragment';
import { render } from '../../../../../testUtils/testUtils';
import { RegistrationIndividualBioData } from './RegistrationIndividualBioData';

describe('components/rdi/details/individual/RegistrationIndividualBioData', () => {
  it('should render', () => {
    const { container } = render(
      <RegistrationIndividualBioData
        individual={fakeImportedIndividualDetailedFragment}
        choicesData={fakeHouseholdChoices}
      />,
    );
    expect(container).toMatchSnapshot();
  });
});
