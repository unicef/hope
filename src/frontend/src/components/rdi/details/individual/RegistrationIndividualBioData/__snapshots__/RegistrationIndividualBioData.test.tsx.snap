// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/rdi/details/individual/RegistrationIndividualBioData should render 1`] = `
<div>
  <div
    class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 sc-khLCKb iJuZvT css-1ps6pg7-MuiPaper-root"
  >
    <div
      class="sc-fAUdSK cpcECW"
    >
      <h6
        class="MuiTypography-root MuiTypography-h6 css-1j53xw6-MuiTypography-root"
      >
        Bio Data
      </h6>
    </div>
    <div
      class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-6 css-zow5z4-MuiGrid-root"
    >
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-gtLWhw bkWrmx"
            color="textSecondary"
          >
            Full Name
          </span>
          <div
            data-cy="label-Full Name"
          >
            <span
              class="sc-egkSDF XYdBd"
              color="textSecondary"
            >
              Alicja Kowalska
            </span>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-gtLWhw bkWrmx"
            color="textSecondary"
          >
            Given Name
          </span>
          <div
            data-cy="label-Given Name"
          >
            <span
              class="sc-egkSDF XYdBd"
              color="textSecondary"
            >
              Alicja
            </span>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-gtLWhw bkWrmx"
            color="textSecondary"
          >
            Middle Name
          </span>
          <div
            data-cy="label-Middle Name"
          >
            -
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-gtLWhw bkWrmx"
            color="textSecondary"
          >
            Family Name
          </span>
          <div
            data-cy="label-Family Name"
          >
            <span
              class="sc-egkSDF XYdBd"
              color="textSecondary"
            >
              Kowalska
            </span>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-gtLWhw bkWrmx"
            color="textSecondary"
          >
            Gender
          </span>
          <div
            data-cy="label-Gender"
          >
            <span
              class="sc-egkSDF XYdBd"
              color="textSecondary"
            >
              Female
            </span>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-gtLWhw bkWrmx"
            color="textSecondary"
          >
            Age
          </span>
          <div
            data-cy="label-Age"
          >
            <span
              class="sc-egkSDF XYdBd"
              color="textSecondary"
            >
              80
            </span>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-gtLWhw bkWrmx"
            color="textSecondary"
          >
            Date of Birth
          </span>
          <div
            data-cy="label-Date of Birth"
          >
            <span
              class="sc-egkSDF XYdBd"
              color="textSecondary"
            >
              <time
                datetime="-894672000000"
              >
                26 Aug 1941
              </time>
            </span>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-gtLWhw bkWrmx"
            color="textSecondary"
          >
            Estimated Date of Birth
          </span>
          <div
            data-cy="label-Estimated Date of Birth"
          >
            <span
              class="sc-egkSDF XYdBd"
              color="textSecondary"
            >
              No
            </span>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-gtLWhw bkWrmx"
            color="textSecondary"
          >
            Marital Status
          </span>
          <div
            data-cy="label-Marital Status"
          >
            <span
              class="sc-egkSDF XYdBd"
              color="textSecondary"
            >
              Married
            </span>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-gtLWhw bkWrmx"
            color="textSecondary"
          >
            Work Status
          </span>
          <div
            data-cy="label-Work Status"
          >
            <span
              class="sc-egkSDF XYdBd"
              color="textSecondary"
            >
              Not provided
            </span>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-gtLWhw bkWrmx"
            color="textSecondary"
          >
            Pregnant
          </span>
          <div
            data-cy="label-Pregnant"
          >
            <span
              class="sc-egkSDF XYdBd"
              color="textSecondary"
            >
              Yes
            </span>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-gtLWhw bkWrmx"
            color="textSecondary"
          >
            Household ID
          </span>
          <div
            data-cy="label-Household ID"
          >
            <span
              class="sc-egkSDF XYdBd"
              color="textSecondary"
            >
              -
            </span>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-gtLWhw bkWrmx"
            color="textSecondary"
          >
            Role
          </span>
          <div
            data-cy="label-Role"
          >
            <span
              class="sc-egkSDF XYdBd"
              color="textSecondary"
            >
              Alternate collector
            </span>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-gtLWhw bkWrmx"
            color="textSecondary"
          >
            Relationship to HOH
          </span>
          <div
            data-cy="label-Relationship to HOH"
          >
            <span
              class="sc-egkSDF XYdBd"
              color="textSecondary"
            >
              Not a Family Member. Can only act as a recipient.
            </span>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-gtLWhw bkWrmx"
            color="textSecondary"
          >
            Preferred language
          </span>
          <div
            data-cy="label-Preferred language"
          >
            -
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-1idn90j-MuiGrid-root"
      >
        <div
          class="sc-dstKZu kpmDQU"
        />
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-gtLWhw bkWrmx"
            color="textSecondary"
          >
            Observed disabilities
          </span>
          <div
            data-cy="label-Observed disabilities"
          >
            -
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-gtLWhw bkWrmx"
            color="textSecondary"
          >
            Seeing disability severity
          </span>
          <div
            data-cy="label-Seeing disability severity"
          >
            <span
              class="sc-egkSDF XYdBd"
              color="textSecondary"
            >
              None
            </span>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-gtLWhw bkWrmx"
            color="textSecondary"
          >
            Hearing disability severity
          </span>
          <div
            data-cy="label-Hearing disability severity"
          >
            <span
              class="sc-egkSDF XYdBd"
              color="textSecondary"
            >
              None
            </span>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-gtLWhw bkWrmx"
            color="textSecondary"
          >
            Physical disability severity
          </span>
          <div
            data-cy="label-Physical disability severity"
          >
            <span
              class="sc-egkSDF XYdBd"
              color="textSecondary"
            >
              None
            </span>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-gtLWhw bkWrmx"
            color="textSecondary"
          >
            Remembering or concentrating disability severity
          </span>
          <div
            data-cy="label-Remembering or concentrating disability severity"
          >
            <span
              class="sc-egkSDF XYdBd"
              color="textSecondary"
            >
              None
            </span>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-gtLWhw bkWrmx"
            color="textSecondary"
          >
            Self-care disability severity
          </span>
          <div
            data-cy="label-Self-care disability severity"
          >
            <span
              class="sc-egkSDF XYdBd"
              color="textSecondary"
            >
              None
            </span>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-gtLWhw bkWrmx"
            color="textSecondary"
          >
            Communicating disability severity
          </span>
          <div
            data-cy="label-Communicating disability severity"
          >
            <span
              class="sc-egkSDF XYdBd"
              color="textSecondary"
            >
              None
            </span>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-gtLWhw bkWrmx"
            color="textSecondary"
          >
            Disability
          </span>
          <div
            data-cy="label-Disability"
          >
            <span
              class="sc-egkSDF XYdBd"
              color="textSecondary"
            >
              Not Disabled
            </span>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-1idn90j-MuiGrid-root"
      >
        <div
          class="sc-dstKZu kpmDQU"
        />
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div
          class="MuiBox-root css-cgq59l"
        >
          <div
            class="MuiBox-root css-1itv5e3"
          >
            <div>
              <span
                class="sc-gtLWhw bkWrmx"
                color="textSecondary"
              >
                National ID
              </span>
              <div
                data-cy="label-National ID"
              >
                <span
                  class="sc-egkSDF XYdBd"
                  color="textSecondary"
                >
                  WSH221365
                </span>
              </div>
            </div>
          </div>
          <div>
            <span
              class="sc-gtLWhw bkWrmx"
              color="textSecondary"
            >
              issued
            </span>
            <div
              data-cy="label-issued"
            >
              <span
                class="sc-egkSDF XYdBd"
                color="textSecondary"
              >
                Poland
              </span>
            </div>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-1idn90j-MuiGrid-root"
      >
        <div
          class="sc-dstKZu kpmDQU"
        />
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-gtLWhw bkWrmx"
            color="textSecondary"
          >
            Email
          </span>
          <div
            data-cy="label-Email"
          >
            -
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-gtLWhw bkWrmx"
            color="textSecondary"
          >
            Phone Number
          </span>
          <div
            data-cy="label-Phone Number"
          >
            <span
              class="sc-egkSDF XYdBd"
              color="textSecondary"
            >
              +48503123555
            </span>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-gtLWhw bkWrmx"
            color="textSecondary"
          >
            Alternate Phone Number
          </span>
          <div
            data-cy="label-Alternate Phone Number"
          >
            <span
              class="sc-egkSDF XYdBd"
              color="textSecondary"
            >
              -
            </span>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
